addon/components/polaris-text-field.js

```diff
@@ -403,6 +463,15 @@ export default Component.extend(
       }
     ).readOnly(),
 
+    inputClassName: computed('suffix', function() {
+      let classes = ['Polaris-TextField__Input'];
+      if (this.get('suffix')) {
```

Comment:
Why not use an if helper in the template?

Reply:
if there's a reason, I guess it's because React implementation has it like this....but yeah, we can totally do that, just wondering if more things will be added here in the future or not

Reply:
This follows suit with how we apply classes to all our other components, I think it would be fine to keep this as-is.